public/app/features/dashboard/dashgrid/DashboardEmpty.tsx

```diff
@@ -1,4 +1,5 @@
 import { css } from '@emotion/css';
```

Comment:
This file got a little bit awkward to review. Let's extract the hooks into a separate file and keep each component into their own files. You can have a folder `DashboardEmpty` that contains all these files so we can have them grouped for maintenance purposes.

Reply:
👍 will move them out